caddytls/ctmonitor.go

```diff
@@ -0,0 +1,298 @@
+package caddytls
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/mholt/caddy"
+)
+
+func init() {
+	caddy.RegisterEventHook("ctmonitor", startMonitoring)
+}
+
+const (
+	certSpotterAPIBase string = "https://api.certspotter.com/v1/issuances"
+)
+
+var (
+	filePath     string = filepath.Join(caddy.AssetsPath(), "ct_id")
+	ctConfigFile string = filepath.Join(caddy.AssetsPath(), "ct_config")
+)
+
+// CtConfig struct will allow me to get the config data from a file
+type CtConfig struct {
+	IncludeSubdomains bool `json:"subdomains"`
+	IncludeWildCards  bool `json:"wildCards"`
```

Comment:
Make the JSON fields match the field name: `include_subdomains` and `include_wildcards`